### PR TITLE
feat(FN-3449): utilisation report premium payments tab copy update

### PR DESCRIPTION
**Pull request by Zainzzkk**
## Introduction :pencil2:
This PR updates the content on the utilisation report premium payments section

## Resolution :heavy_check_mark:
* Copy change
* Added data-cy
* Updated test

### CODE REVIEW

**Review by ttbarnes — APPROVED** [review]
Great work

File: trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk [hunk 1]
```diff
@@ -34,11 +34,7 @@
 
     <div class="govuk-grid-row premium-payments-description-filter-container">
       <div class="govuk-grid-column-one-half premium-payments-description-container">
-        <p class="govuk-body">Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button.</p>
-
-        <p class="govuk-body">
-          When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected.
-        </p>
+        <p class="govuk-body" data-cy="received-payments-text">Received payments are entered against reported fees. When payments show as matched, the adjustment data for keying into ACBS will be automatically generated.</p>
```
Comment (RLavender98) [inline trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk:37]: Should this change not be for only the read only users with PDC_RECONCILE still seeing the same text?

**Review by RLavender98 — CHANGES_REQUESTED** [review]

File: trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk [hunk 2]
```diff
@@ -34,11 +34,15 @@
...
     <div class="govuk-grid-row premium-payments-description-filter-container">
       <div class="govuk-grid-column-one-half premium-payments-description-container">
-        <p class="govuk-body">Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button.</p>
+        {% if userCanEdit %}
+          <p class="govuk-body" data-cy="received-payments-entered-text">Received payments are entered against reported fees through selection and then selection of the 'Add a payment' button.</p>
 
-        <p class="govuk-body">
-          When payments show as matched, the adjustment data for keying into ACBS will be automatically generated when the 'Generate keying data' button is selected.
-        </p>
+          <p class="govuk-body" data-cy="when-payments-show-text">
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply trade-finance-manager-ui/templates/utilisation-reports/utilisation-report-reconciliation-for-report.njk]: done

File: trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts [hunk 3]
```diff
@@ -97,6 +97,19 @@ describe(page, () => {
...
+  it('should render the premium payments with the relevant text for non-PDC_RECONCILE users', () => {
+    const wrapper = getWrapper({ ...params, user: aPdcReadUser() });
+    const premiumPaymentsTabSelector = 'div#premium-payments';
+
+    wrapper.expectElement(premiumPaymentsTabSelector).toExist();
+
+    wrapper
+      .expectText(`[data-cy="received-payments-text"]`)
+      .toMatch(
+        /Received payments are entered against reported fees. When payments show as matched, the adjustment data for keying into ACBS will be automatically generated./,
```
[the remark this answers is not in the document]
Comment (Zainzzkk) [reply trade-finance-manager-ui/component-tests/utilisation-reports/utilisation-report-reconciliation-for-report.component-test.ts:111]: added

**Review by RLavender98 — APPROVED** [review]

**Pull request merged**